ci/axis/rapidsai-driver.yml

```diff
@@ -24,6 +23,7 @@ IMAGE_TYPE:
 
 LINUX_VER:
   - centos7
+  - ubuntu20.04
```

Comment:
I believe you need to also update the `DOCKER_FILE` axis and add some excludes.

Reply:
whoops. good catch. addressed in d9e85c7